setup.py

```diff
@@ -144,5 +144,9 @@ def pkgPath(root, path, rpath="/"):
     scripts=['bin/diamond', 'bin/diamond-setup'],
     data_files=data_files,
     install_requires=install_requires,
+    classifiers=[
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        ],
```

Comment:
hmm interesting
it doesn't complain [here](https://github.com/nickjacobson/Diamond/blob/c1e74905303bc3f4d6d2ce191212b2c923e14694/setup.py#L36) or [here](https://github.com/nickjacobson/Diamond/blob/c1e74905303bc3f4d6d2ce191212b2c923e14694/setup.py#L43)